scripts/registry.sh

```diff
@@ -64,12 +64,16 @@ publish() {
     yarn build
     git commit --allow-empty -am "chore: prepare for publishing"
 
+    yarn lerna run build:types
```

Comment:
In the sync, I fixed this in one place but not here. In the future, grepping for build:types and clean:types should be sufficient to be sure we update everywhere it’s needed, but I wonder if there’s anywhere we can leave a note for our future selves where we’d be likely to find it when we needed it. I don’t have a suggestion.